packages/coil-extension/src/popup/components/RandomThankYouMessage.tsx

```diff
@@ -17,11 +17,21 @@ const RandomMessageWrapper = styled('div')({
 export const RandomThankYouMessage = () => {
   const [random, setRandom] = useState<number>(0)
 
+  // const messagesArray = [
```

Comment:
Is there a reason to keep these around?

Reply:
Yeah, it looks like fabi is still working on possible messages. Since they are still WIP I just added the new ones to a new list since we don't know what the final list looks like right now.